engine/components-api/src/com/cloud/network/vpc/VpcManager.java

```diff
@@ -161,8 +161,9 @@
      * @param gateway
      * @param networkOwner TODO
      */
-        void
-        validateNtwkOffForNtwkInVpc(Long networkId, long newNtwkOffId, String newCidr, String newNetworkDomain, Vpc vpc, String gateway, Account networkOwner, Long aclId);
+    void validateNtwkOffForNtwkInVpc(Long networkId, long newNtwkOffId, String newCidr, String newNetworkDomain, Vpc vpc, String gateway, Account networkOwner, Long aclId);
 
     List<PrivateGateway> getVpcPrivateGateways(long vpcId);
+
+    CIDR[] getZoneSuperCidrList(long zoneId) throws BadCIDRException;
```

Comment:
Why is the list of super CIDRs being represented as an array instead of a ``List``, ``Set``, or custom collected type for ``CIDR``s?